tests.py

```diff
@@ -170,7 +170,7 @@ def test_advanced_add(self):
         with open(self.config_file) as f:
             # check that property is really flushed out to the config?
             content = f.read().encode('ascii')
-            self.assertIn(b'identityfile "/tmp/idfilecheck.rsa"', content)
+            self.assertIn(b'IdentityFile "/tmp/idfilecheck.rsa"', content)
             self.assertIn(b"stricthostkeychecking yes", content)
```

Comment:
Hmm, maybe so.  Those look to be coming from `--o "StrictHostKeyChecking=yes` above...  which I guess means something like "Add this key val pair".

But really the only part of this project I actually care about is this one file:  `storm/parsers/ssh_config_parser.py`.  Maybe I should just break that file out into separate micro-repo and close this PR.